extra/2-piping.js

```diff
@@ -16,26 +16,27 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
-
+function format(num) {
+  return "£" + num;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = "£" + ((startingValue + 10) * 2);
 
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = "£" + ((startingValue + 10) * 2);
```

Comment:
```let goodCode = format(multiply(add(startingValue, 10) * 2))```
Why is this better? Because it is using functions that we have already declared to carry out the work rather that writing new code to achieve the same thing. Is it more readable, I would say maybe piping more that 3 functions like this starts to get silly.